prisma/schema.prisma

```diff
@@ -112,3 +113,23 @@ model Attendees {
   attendeeId Int
   attendee   Users      @relation(fields: [attendeeId], references: [id])
 }
+
+enum TokenType {
+  GOOGLECAL
+  OUTLOOKCAL
+}
+
+model AccessToken {
+  id              Int       @id @default(autoincrement())
+  user            Users     @relation(fields: [userId], references: [id])
+  userId          Int
+  calendarType    TokenType
+  associatedEmail String    @unique
```

Comment:
We can have multiple tokens of the same user. Thus, there will be multiple `associatedEmail` in this column. In that case, this wont be marked as unique. The associated email will be the email associated with the calendar. User also might authorise a calendar of another email via their account.